librz/il/_ct_opcodes.h

```diff
@@ -0,0 +1,156 @@
+#ifndef BUILD__CT_OPCODES_H
+#define BUILD__CT_OPCODES_H
+
+#include "definitions/wrapper.h"
+
+// All `ret` members are index to their type-specific array
```

Comment:
Convert to Doxygen

Reply:
Working on adding doxygen comments to structure in header files.